lca-cli/postpivot/postpivot.go

```diff
@@ -478,6 +479,47 @@ func (p *PostPivot) setNodeIPIfNotProvided(ctx context.Context,
 	return nil
 }
 
+func (p *PostPivot) setNodeIPHint(ctx context.Context, nodeIp, nodeIPConfFile string) error {
+	if nodeIp == "" {
+		// TODO: maybe better to delete nodeip hint file in this case?
+		p.log.Info("Node ip was not provided, skipping node ip hint changes")
+		return nil
+	}
+	var hintSubnet net.IP
+	p.log.Infof("Searching for subnet that matches %s", nodeIp)
+	// retry is required as installation service runs before
+	// network online and ip probably will not be configured on first run
+	_ = wait.PollUntilContextCancel(ctx, time.Second, true, func(ctx context.Context) (done bool, err error) {
+		addresses, err := p.ops.ListNodeAddresses()
+		if err != nil {
+			p.log.Warnf("failed to list node addresses, will retry. err: %s", err.Error())
+			return false, nil
+		}
+
+		for _, addr := range addresses {
+			ip, cidr, _ := net.ParseCIDR(addr.String())
```

Comment:
```suggestion
			ip, subnet, _ := net.ParseCIDR(addr.String())
```

Reply:
actually it is cidr so not sure why you want it as subnet. I believe better to stick to the same naming as in assisted